pulp_ansible/app/tasks/collections.py

```diff
@@ -640,7 +642,7 @@ async def _add_collection_version(self, api_version, collection_version_url, met
         await self.parsing_metadata_progress_bar.aincrement()
         await self.put(d_content)
 
-        if signatures:
+        if signatures or marks:
             collection_version = await d_content.resolution()
```

Comment:
ohh, (not really a comment to this pr) this can significantly slow down sync, as this collection version basically jumps the whole pipeline without waiting to be processed in batches. We should refactor this soon.

Reply:
lets tag @gerrod3 for a discussion on this topic on our next sync meeting

Reply:
Please add some meaningful sync tests. We want to be sure we do not regress when we refactor this.